components/factory/forms/BurnForm.tsx

```
@@ -89,201 +113,245 @@
     } finally {
       setIsSigning(false);
     }
   };
 
   const handleMultiBurn = async () => {
-    if (
-      burnPairs.some(
-        (pair) => !pair.address || !pair.amount || isNaN(Number(pair.amount)),
-      )
-    ) {
+    if (burnPairs.some(pair => !pair.address || !pair.amount || isNaN(Number(pair.amount)))) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)